cnf-certification-test/platform/suite.go

```diff
@@ -114,8 +118,30 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 			ginkgo.Skip(" non ocp cluster ")
 		}
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestServiceMeshIdentifier)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.DebugPods)
+		TestServiceMesh(&env)
+	})
 })
 
+func TestServiceMesh(env *provider.TestEnvironment) {
+	var badPods []string
+	for _, put := range env.Pods {
+		data := put.Containers
+		fmt.Println(data)
+		for _, d := range data {
+			if d.Status.Name == istio {
+				tnf.ClaimFilePrintf("For pods %s ,ns %s we have service mesh", put.Data.Name, put.Data.Namespace)
+				continue
+			}
+			badPods = append(badPods, put.Data.Name)
```

Comment:
same here u are adding the badpods per container, is it ok?

Reply:
yes, I was save all the containers in any pods that no have "Istio". I will update the message if I found Istio , so I adding details about which container.